src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ReviewEvidenceNote.cshtml

```diff
@@ -33,20 +31,46 @@
         @Html.Partial("_ViewEvidenceNotePartial", Model.ViewEvidenceNoteViewModel)
 
         <!-- radiobuttons for approved -->
-        <p>Choose whether to approve, reject or return the evidence note</p>
+        <p class="govuk-!-margin-top-7">Choose whether to approve, reject or return the evidence note</p>
         <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)" id="approval-options">
             @Html.Gds().ValidationMessageFor(m => m.SelectedValue)
             @this.WeeeGds().RadioButtonsFor(
                 m => m.SelectedValue,
                 Model.PossibleValues,
-                "Choose whether to approve, reject or return the evidence note",
-                RadioButtonLegend.VisuallyHidden,
+                " ",
+                RadioButtonLegend.Displayed,
                 RadioButtonLayout.Stacked,
                 Model.HintItems)
         </div>
 
+        <div id="reason-text" style="display: none;" class="govuk-!-margin-bottom-7">
```

Comment:
The problem with display:none by default is that it will be hidden for people who have javascript turned off.
An alternative way is for example,

This div will always be displayed...
 <div id="mydiv">
            <p>test</p>
        </div>


Then it can be hidden for javascript people by putting the following at the bottom of the cshtml file
@section scripts {
    <script type="text/javascript">
        $(document).ready(function () {
            document.getElementById('mydiv').style.display = "none";
        });
    </script>
}